README.md

```diff
@@ -12,6 +12,11 @@ Create migration
 $ rails generate mongoid:migration <your_migration_name_here>
 ```
 
+You can also create migration for shards with:
+```
+$ rails generate mongoid:migration <your_migration_name_here> --shards
+```
+
```

Comment:
That's a good start but we still need some explanation about what is sharding in this gem and how it works IMO